docker/Dockerfile.demo

```diff
@@ -0,0 +1,55 @@
+FROM ubuntu:xenial
+
+RUN apt-get update --fix-missing
+
+RUN apt-get -y install curl
+
+# Add the Cloud SDK distribution URI as a package source
+RUN echo "deb http://packages.cloud.google.com/apt cloud-sdk-xenial main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+
+# Import the Google Cloud Platform public key
+RUN curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+
+# install node
+RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
```

Comment:
I know that this is exactly how Nodesource instructs you to set this up, but I think if you use the docker `ADD` instruction it will detect updates to that file and then re-run this step if it has changed. I'm not sure if that's preferred.

Reply:
@bjester thank you for the suggestion! I will definitely try the new way :)

Reply:
Oh I didn't know you can add from a remote URL! Nice